sql/5_0_0-to-5_0_1_upgrade.sql

```diff
@@ -95,3 +95,162 @@
+  `last_level_billed` int  NOT NULL DEFAULT 0 COMMENT '0=none, 1=ins1, 2=ins2, etc',
+  `last_level_closed` int  NOT NULL DEFAULT 0 COMMENT '0=none, 1=ins1, 2=ins2, etc',
+  `last_stmt_date`    date DEFAULT NULL,
+  `stmt_count`        int  NOT NULL DEFAULT 0,
+  `provider_id` INT(11) DEFAULT '0' COMMENT 'default and main provider for this visit',
+  `supervisor_id` INT(11) DEFAULT '0' COMMENT 'supervising provider, if any, for this visit',
+  `invoice_refno` varchar(31) NOT NULL DEFAULT '',
+  `referral_source` varchar(31) NOT NULL DEFAULT '',
+  `billing_facility` INT(11) NOT NULL DEFAULT 0,
+  `external_id` VARCHAR(20) DEFAULT NULL,
+  `pos_code` tinyint(4) default NULL,
+  `counselors` VARCHAR (255),
+  `appt_id` INT(11) default NULL,
+  PRIMARY KEY  (`id`),
+  KEY `pid_encounter` (`group_id`, `encounter`),
+  KEY `encounter_date` (`date`)
+) ENGINE=InnoDB AUTO_INCREMENT=1 ;
+#EndIf
+
+#IfMissingColumn forms therapy_group_id
+ALTER TABLE  `forms` ADD  `therapy_group_id` INT(11) DEFAULT NULL;
+#EndIf
+
+#IfMissingColumn registry patient_encounter
+ALTER TABLE `registry` ADD `patient_encounter` TINYINT NOT NULL DEFAULT '1';
+#EndIf
+
+#IfMissingColumn registry therapy_group_encounter
+ALTER TABLE `registry` ADD `therapy_group_encounter` TINYINT NOT NULL DEFAULT '0';
+#EndIf
+
+
+#IfNotRow2D list_options list_id lists option_id attendstat
+INSERT INTO list_options (`list_id`, `option_id`, `title`, `seq`, `is_default`, `option_value`) VALUES ('lists', 'attendstat', 'Group Attendance Statuses', '15', '0', '0');
+INSERT INTO list_options (`list_id`, `option_id`, `title`, `seq`, `is_default`, `option_value`, `notes`, `toggle_setting_1`) VALUES ('attendstat', '-', '- Not Reported', '10', '0', '0', 'FEFDCF|0', '0');
+INSERT INTO list_options (`list_id`, `option_id`, `title`, `seq`, `is_default`, `option_value`, `notes`, `toggle_setting_1`) VALUES ('attendstat', '@', '@ Attended', '20', '0', '0', 'FF2414|0', '1');
+INSERT INTO list_options (`list_id`, `option_id`, `title`, `seq`, `is_default`, `option_value`, `notes`, `toggle_setting_1`) VALUES ('attendstat', '?', '? Did Not Attend', '30', '0', '0', 'BFBFBF|0', '1');
+INSERT INTO list_options (`list_id`, `option_id`, `title`, `seq`, `is_default`, `option_value`, `notes`, `toggle_setting_1`) VALUES ('attendstat', '~', '~ Late Arrival', '40', '0', '0', 'BFBFBF|0', '1');
+INSERT INTO list_options (`list_id`, `option_id`, `title`, `seq`, `is_default`, `option_value`, `notes`, `toggle_setting_1`) VALUES ('attendstat', 'x', 'x Cancelled', '50', '0', '0', 'FEFDCF|0', '0');
+#EndIf
+
+#IfNotRow2D registry directory group_attendance category Clinical
+INSERT INTO registry (name, state, directory, id, sql_run, unpackaged, date, priority, category, nickname, patient_encounter, therapy_group_encounter) VALUES ('Group Attendance Form', 1, 'group_attendance', 22, 1, 1, '2015-10-15 00:00:00', 0, 'Clinical', '',0,1);
```

Comment:
remove id from this. it is autoincremented and will be set automatically. thus will avoid conflicts by not setting it.